Switcheroo/MainWindow.xaml.cs

```diff
@@ -109,6 +109,76 @@ private void SetUpKeyBindings()
                     tb.IsEnabled = true;
                     tb.Focus();
                 }
+                else if (args.SystemKey == Key.D1 && Keyboard.Modifiers.HasFlag(ModifierKeys.Alt))
+                {
+                    lb.SelectedIndex = 0;
+                    ScrollSelectedItemIntoView();
+                    Switch();
+                    HideWindow();
+                }
+                else if (args.SystemKey == Key.D2 && Keyboard.Modifiers.HasFlag(ModifierKeys.Alt))
+                {
+                    lb.SelectedIndex = 1;

```

Comment:
Could you extract a method and reuse it to avoid code duplication?

Reply:
Done.

Reply:
Done